Makefile

```diff
@@ -140,6 +140,10 @@ bash: compose_build
 refresh: kill compose_build
 	$(COMPOSE) run -e SKIP_DUMMY=$$SKIP_DUMMY app bash -c '$(WAIT_FOR_DB) $(PYTHON_MIGRATE)&&$(LOAD_LOCALES)&&$(LOAD_COUNTRIES)&&$(LOAD_DUMMY_EXPERIMENTS)'
 
+dependabot_approve:
+	echo "Install and configure the Github CLI https://github.com/cli/cli"
+	gh pr list | grep "chore(deps" | awk '{print $$1}' | xargs -n1 gh pr review -a -b "@dependabot squash and merge"
```

Comment:
Fascinating!
https://cli.github.com/manual/gh_pr_list

Looks like instead of the `grep "chore(deps"`, we could possibly do a safer `--author` query:

```sh
gh pr list --author app/dependabot
```

I was also trying to play with the `--json` and `--jq` flags to see if you could avoid `awk`, but not sure this is any better or nein.

```sh
gh pr list --author app/dependabot --state=closed --json number --jq ".[] | .number"
```